src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient.swift

```diff
@@ -23,18 +23,23 @@ final class HTTPClient: Client {
 	// MARK: Creating
 	init(
 		configuration: Configuration,
+		packageVerifier: @escaping SAPDownloadedPackage.Verification = SAPDownloadedPackage.Verifier().verify,
```

Comment:
This is perhaps the most impactful change. 
In order to make unit testing possible for the signature verification, or to skip verification if a unit test wants to, we need a method to extract the verification process from the `HTTPClient`.

I thought passing in a closure for the verification process is a good way of doing this. It's defaulted for the production version but can be replaced as required for a unit test.

Reply:
very nice.